realsense2_camera/src/profile_manager.cpp

```diff
@@ -160,6 +160,21 @@ void ProfilesManager::addWantedProfiles(std::vector<rs2::stream_profile>& wanted
             ROS_DEBUG_STREAM("Found profile for " << ros_stream_to_string(sip.first) << ":" << sip.second);
         }
     }
+
+    // Warn the user if the enabled stream cannot be opened due to wrong profile selection
+    for (auto const & x : _enabled_profiles)
```

Comment:
During enabling / re-enabling the stream, warn the user if the enabled stream cannot be opened due to wrong profile selection.

Reply:
would prefer to rename "x" to something more understandable (like "profile")